manifests/localzone.pp

```diff
@@ -26,15 +26,29 @@
 # @param template_name Use a custom template.
 #
 define unbound::localzone (
-  Unbound::Local_zone_type $type,
-  String $zone = $name,
-  $config_file = $unbound::config_file,
-  Array[Unbound::Resource_record_type] $local_data = [],
-  String $template_name = 'unbound/local_zone.erb'
+  Unbound::Local_zone_type             $type,
+  String                               $zone          = $name,
+  Stdlib::Absolutepath                 $config_file   = $unbound::config_file,
+  String                               $template_name = 'unbound/local_zone.erb',
+  Array[Unbound::Resource_record_type] $local_data    = [],
 ) {
+  $records = $local_data.map |$record| {
```

Comment:
there's a lot going on here. Can you please add some comments about what's happening?

Reply:
Fair comment will do

Reply:
Updated let me know what's missing?